angular/scripts/publish.sh

```diff
@@ -1,10 +1,19 @@
 #!/usr/bin/env bash
 
+branch=master # default
 MASTER_VERSION=`node -p "require('./package.json').version"`
 NPM_LATEST_VERSION=`npm show @pxblue/angular-themes version`
 NPM_BETA_VERSION=`npm show @pxblue/angular-themes@beta version`
 NPM_ALPHA_VERSION=`npm show @pxblue/angular-themes@alpha version`
 
+# Parse the branch from the -b flag
+while getopts b: flag
+do
+    case "${flag}" in
+        b) branch=${OPTARG};;
```

Comment:
b) 

What is this?

Reply:
It's a bash-syntax switch statement for parsing command line options (`-b`)